### PR TITLE
[1LP][RFR] Updating fauxfactor to 3.0.6

### PR DESCRIPTION
**Pull request by jawatts**

### CODE REVIEW

File: requirements/frozen.py3.txt [hunk 1]
```diff
@@ -132,7 +132,7 @@ docutils==0.14
 dogpile.cache==0.7.1
 dump2polarion==0.39.1
 entrypoints==0.3
-fauxfactory==3.0.2
+fauxfactory==3.6.0
```
Comment (john-dupuy) [inline requirements/frozen.py3.txt]: :-1:

File: requirements/frozen_docs.py3.txt [hunk 2]
```diff
@@ -131,7 +131,7 @@ docutils==0.14
 dogpile.cache==0.7.1
 dump2polarion==0.39.1
 entrypoints==0.3
-fauxfactory==3.0.2
+fauxfactory==3.6.0
```
Comment (john-dupuy) [inline requirements/frozen_docs.py3.txt]: :-1:

File: requirements/frozen.py3.txt (hunk 1 above)
Comment (mshriver) [reply requirements/frozen.py3.txt]: noob

**Comment by jawatts** [comment]
Updated after @mshriver told me the wrong version :)

**Pull request merged**